addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8026,7 +8026,11 @@ msgctxt "#15064"
 msgid "Apply changes directly"
 msgstr ""
 
-#empty string with id 15065
+#. Warning if no dsp modes are present
+#: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp
+msgctxt "#15065"
+msgid "No audio DSP modes present!"
```

Comment:
Yes I can prevent the open with the use on:

```cpp
void CActiveAEDSP::OnSettingAction(const CSetting *setting)
{
  if (setting == NULL)
    return;

  const std::string &settingId = setting->GetId();

  if (settingId == CSettings::SETTING_AUDIOOUTPUT_DSPSETTINGS)
  {
    if (IsActivated())
    {
      CGUIDialogAudioDSPManager *dialog = (CGUIDialogAudioDSPManager *)g_windowManager.GetWindow(WINDOW_DIALOG_AUDIO_DSP_MANAGER);
      if (dialog)
        dialog->Open();
    }
  }
  else if (settingId == CSettings::SETTING_AUDIOOUTPUT_DSPRESETDB)
  {
    if (HELPERS::ShowYesNoDialogLines(CVariant{19098}, CVariant{36440}, CVariant{750}) ==
      DialogResponse::YES)
    {
      CDateTime::ResetTimezoneBias();
      ResetDatabase();
    }
  }
}
```

And already planing to hide the button on Estuary if no modes are available.